README.md

```diff
@@ -1,19 +1,17 @@
 # MLH/mlh-site-boilerplate
 
 This is the boilerplate that [Major League Hacking (MLH)][mlh] uses in the
-development of static websites.  It combines [Jekyll][jekyll] with
-[Gulp][gulp] to speed up common development tasks.
+development of static websites.  It uses [Webpack][webpack] to automate and speed up common development tasks.
 
 ## Setup
 
 Before you start, make sure you have [npm][npm-install] installed and the
-relevant version of Ruby (probably using [rvm][rvm]).
+relevant version of Ruby (probably using [rvm][rvm] or [rbenv][rbenv]).
```

Comment:
If this requires ruby, shouldn't we keep the `.ruby-version` file? Or does it not matter for some reason?

Also, we weren't able to move to the node version of libsass?